examples/example_plugin/example_plugin/urls.py

```diff
@@ -45,4 +47,9 @@
         name="examplemodel_changelog",
         kwargs={"model": ExampleModel},
     ),
+    path(
+        "docs/",
+        RedirectView.as_view(url=f"{settings.STATIC_URL}example_plugin/docs/index.html"),
+        name="docs",
+    ),
```

Comment:
This only works for `docs/` right, not something like `docs/models/example_model/`?

Reply:
I am sure you can put as many urls as you want, but it does a redirect, so the address bar will change. Alternatively, we could have Django serve the docs, but that is an antipattern for django since static directory gets served through the webserver, not Django.

Reply:
This should be fine, thanks.